src/test/FreeC/Frontend/Haskell/Tests.hs

```diff
@@ -0,0 +1,11 @@
+-- | Test group for tests of modules below @FreeC.Frontend.Haskell@.
```

Comment:
This formulation is probably used for many test modules, but I don't think `below` fits here, as I couldn't find a dictionary mentioning affiliation as one of the meanings of this word. You could use `of`, `belonging to` or something else instead.

Reply:
See 76f018e97820f0fffe9b197b6a67be1eeb34f2ee.